templates/accounts/volunteer_profile.html

```diff
@@ -147,7 +147,7 @@ <h2 class="font-weight-bolder">Complete Your Profile</h2>
                 <!-- Add profile picture -->
                 <div class="mb-3">
                     <label for="profileImage">Profile Picture <span class="text-muted">(Optional)</span></label>
-                    <input name="profile_image" type="file" id="profileImage" class="form-control-file">
+                    <input name="profile_image" type="file" id="profileImage" class="form-control-file" onchange="return ImageValidation()">
```

Comment:
No need to add `return` here. You just want to execute the function when onChange event is fired and not return anything. Return is important when this function is called on `onsubmit` event of form element.